doc/examples/01_apply_active_class_to_whole_tree.md

```diff
@@ -5,6 +5,13 @@ By default the active menu item is applied with a `current` class and it's ances
 class. To apply a class other than the default or apply the same class to both you can either pass it as option to the 
 render method of your used renderer:
 
+```php
+<?php
+$renderer->render($item, ['currentClass' => 'active', 'ancestor_class' => 'active']);
+```
+
+Old deprecated way:
```

Comment:
No need to document a deprecated way. The only purpose of the deprecated way is to keep BC.

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/351/commits/d7a33998c36319d45c2c52605c16145a2baeff13